vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -66,6 +66,36 @@ type Serializer struct {
 var _ runtime.Serializer = &Serializer{}
 var _ recognizer.RecognizingDecoder = &Serializer{}
 
+func init() {
```

Comment:
Was this an early aborted attempt?

Reply:
This is restoring and then commenting out upstream additions to json-iter so we trigger a conflict. This will get squashed to our existing carry that moved from json-iter to stdlib